sdk/services/kwentaToken.ts

```diff
@@ -0,0 +1,464 @@
+import { wei } from '@synthetixio/wei';
+import axios from 'axios';
+import { Contract as EthCallContract } from 'ethcall';
+import { ethers } from 'ethers';
+import moment from 'moment';
+import KwentaSDK from 'sdk';
+
+import { DEFAULT_NUMBER_OF_FUTURES_FEE } from 'constants/defaults';
+import { FLEEK_BASE_URL, FLEEK_STORAGE_BUCKET } from 'queries/files/constants';
+import { ContractName } from 'sdk/contracts';
+import { formatTruncatedDuration } from 'utils/formatters/date';
+
+import * as sdkErrors from '../common/errors';
+import ERC20ABI from '../contracts/abis/ERC20.json';
+import KwentaArrakisVaultABI from '../contracts/abis/KwentaArrakisVault.json';
+import MultipleMerkleDistributorABI from '../contracts/abis/MultipleMerkleDistributor.json';
+import RewardEscrowABI from '../contracts/abis/RewardEscrow.json';
+import StakingRewardsABI from '../contracts/abis/StakingRewards.json';
+import SupplyScheduleABI from '../contracts/abis/SupplySchedule.json';
+
+const client = axios.create({
+	baseURL: `${FLEEK_BASE_URL}/${FLEEK_STORAGE_BUCKET}/data/`,
+	timeout: 5000,
+});
+
+export type ClaimParams = [number, string, string, string[], number];
+
+type EpochData = {
+	merkleRoot: string;
+	tokenTotal: string;
+	claims: {
+		[address: string]: {
+			index: number;
+			amount: string;
+			proof: string[];
+		};
+	};
+};
+
+export type EscrowData = {
+	id: number;
+	date: string;
+	time: string;
+	vestable: number;
```

Comment:
Would be good to avoid using number and always stick to Wei for values so we remain consistent throughout.

Reply:
Fair enough.